K2Bridge.Tests.UnitTests/JsonConverters/AggregationContainerConverterTests.cs

```diff
@@ -377,6 +396,41 @@ public class AggregationContainerConverterTests
             },
         };
 
+        private static readonly AggregationContainer ExpectedExtendedStatsAggregationWithoutSigma = new AggregationContainer()
+        {
+            PrimaryAggregation = null,
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
+                {
+                    PrimaryAggregation = new ExtendedStatsAggregation
+                    {
+                        Field = "metric",
+                        Key = "2",
+                    },
+                    SubAggregations = new AggregationDictionary(),
+                },
+            },
+        };
+
+        private static readonly AggregationContainer ExpectedExtendedStatsAggregationWithSigma = new AggregationContainer()
+        {
+            PrimaryAggregation = null,
+            SubAggregations = new AggregationDictionary
+            {
+                ["2"] = new AggregationContainer
+                {
+                    PrimaryAggregation = new ExtendedStatsAggregation
+                    {
+                        Field = "metric",
+                        Key = "2",
+                        Sigma = 3,
```

Comment:
btw, what is sigma?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-extendedstats-aggregation.html#_standard_deviation_bounds

By default, the `extended_stats` metric will return an object called `std_deviation_bounds`, which provides an interval of plus/minus two standard deviations from the mean. 
This can be a useful way to visualize variance of your data. If you want a different boundary, for example three standard deviations, you can set `sigma` in the request